### DISCUSSION TITLE
Allow syncing Overleaf documents to this repo

### DISCUSSION THREAD

**Discussion by zerothi** [dtudk/Admin#3] [Ideas]
@amcar-dtu, you have requested this feature to be enabled.

I would like some more context as to your ideas?

My current understanding is that the sync enables one to synchronize documents from Overleaf to a github site, and that's pretty much it?

Why would you want this, and what would it bring to the DTU environment?

**Comment by amcar-dtu** [comment]
Hi @zerothi,

I'm currently working to some LaTeX documents on Overleaf and I am syncing them with my DTU account on GitHub to have them versioned and to be able to work on them locally on VSCode, so that I can push local changes to GitHub and the fetch them on Overleaf.

So for now I can just decide manually which Overleaf document to synchronise with GitHub, and as per my understanding Overleaf asked me that permission to sync all my Overleaf documents with GitHub.

So if it's a problem I can continue working as now, without that permission, otherwise would be handy to have my Overleaf documents versioned on GitHub.

Let me know what are your thoughts about, and thanks a lot for your help!

**Reply by zerothi** [reply]
Hi, thanks for your feedback.

As such, I think we will strive to make this namespace a "coding environment", i.e. articles and other "non-coding" documents should not be retained here. This is just to have a line of sight where people don't use the dtudk namespace for article writing. 

My proposal would be that you retain your overleaf documents locally (on your account) and then take it from there.

Would this be fine?

**Reply by amcar-dtu** [reply]
That's perfectly fine, thanks for your feedback! ☺️

**Reply by zerothi** [reply]
Great, I'll decline the authorization request for dtudk. Thanks!
